### DISCUSSION TITLE
About the pm.batteryLevel

### DISCUSSION THREAD

**Discussion by seunghyunnn** [bitcraze/discussions#222] [Q&A]
Hi,
I am trying to use the [pm.batteryLevel](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#pmbatterylevel) to log how much battery is left. 

I wrote my python code to fly the crazyflie and used uint8_t type from https://github.com/bitcraze/crazyflie-lib-python/blob/master/cflib/crazyflie/log.py to read the values of pm.batteryLevel, expecting the values to be a value between 0 - 100. 

However, when I got the log, I only got tons of 10's with several 0s like the images below. (The values were either 10 or 0). 

![image](https://user-images.githubusercontent.com/67893627/186586161-20cd3374-0c50-47d3-8c45-5d2f4cec34e9.png)
![image](https://user-images.githubusercontent.com/67893627/186586237-e455dc76-e0fb-4fb8-b3f1-4c143540c272.png)

Are they the correct values? If it is not, how should I revise my code? 
If it is, how can I log the percentage of the battery level, between 0 to 100 using python?

Thank you!

**Comment by krichardsson** [comment]
You can test the behavior of the log variable by [setting up logging in the client](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/plotter_tab/). This will show you what to expect and then you can compare with the values you get in your python script.
